plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -811,10 +811,10 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
     }
 
     @Override
-    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle, Constraint constraint)
+    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         JdbcTableHandle handle = (JdbcTableHandle) tableHandle;
-        return jdbcClient.getTableStatistics(session, handle, constraint.getSummary());
+        return jdbcClient.getTableStatistics(session, handle, handle.getConstraint());
```

Comment:
`handle.getConstraint()` is useless  since we pass whole `handle` too.

let's do minimal change here (pass `Constraint.alwaysTrue` to JdbcClient, adorn with a TODO comment) and let's do base-jdbc changes in a follow-up

- we don't have same backward compatibility expectations for base-jdbc
- JdbcClient has many implementations, so it's better to separate

Reply:
Done!